i18n/test/babel-plugin.js

```diff
@@ -36,4 +42,47 @@ describe( 'babel-plugin', () => {
 			expect( isSameTranslation( a, b ) ).to.be.true();
 		} );
 	} );
+
+	describe( '.getTranslatorComment()', () => {
+		function getCommentFromString( string ) {
+			let comment;
+			traverse( transform( string ).ast, {
+				CallExpression( path ) {
+					comment = getTranslatorComment( path );
+				},
+			} );
+
+			return comment;
+		}
+
+		it( 'should not return translator comment on same line but after call expression', () => {
+			const comment = getCommentFromString( '__( \'Hello world\' ); // translators: Greeting' );
```

Comment:
This seems like a potentially useful variation, or is the intent to match existing behavior in WordPress?